src/flexmeasures_client/client.py

```diff
@@ -22,6 +23,7 @@
 POLLING_TIMEOUT = 200.0  # seconds
 REQUEST_TIMEOUT = 20.0  # seconds
 POLLING_INTERVAL = 10.0  # seconds
+API_VERSIONS_LIST = ("v1", "v1_1", "v1_2", "v1_3", "v2_0", "v3_0")
```

Comment:
Just `("v3_0",)`, please. (The comma is needed to keep this a tuple.) The client lacks most functionality for supporting the older versions, plus: FlexMeasures just dropped support for versions 1 to 2.

Reply:
Thanks, removed the versions that are not supported and fixed the tuple